clang/test/Driver/fp-contract.c

```diff
@@ -238,3 +238,17 @@
 // RUN: %clang -### -fno-unsafe-math-optimizations -funsafe-math-optimizations \
 // RUN: -ffp-contract=off -c %s 2>&1 | FileCheck --check-prefix=CHECK-FPC-OFF %s
 
+// RUN: %clang -### -funsafe-math-optimizations -ffp-contract=off -c %s 2>&1 \
```

Comment:
L161 has similar `-funsafe-math-optimizations -ffp-contract=off` checks. The test can be folded there.

Reply:
I just went ahead and added a bunch of `-Werror` to previous lines in 514d80b4feea3c788c1b0821959e96f63c8b8f3d

Reply:
That introduces a merge conflict for this patch. Is there a way I can solve that without having to do a rebase and force push for this, which kills the ability to see what changed between updates? I guess with a small-ish patch like this that's not a big problem, but it's something I try to avoid in general.

Reply:
Some folks prefer `git merge origin/main` while some just use `git rebase origin/main` and force push.

I am fine with either style, but someone really dislike force push (perhaps not for such a trivial change).